src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -40,11 +40,13 @@ import './header_breadcrumbs.scss';
 interface Props {
   appTitle$: Observable<string>;
   breadcrumbs$: Observable<ChromeBreadcrumb[]>;
+  isDarkMode: boolean | undefined;
```

Comment:
nit - personal preference is for 
```suggestion
  isDarkMode?: boolean;
```
rather than explicit union.